### PR TITLE
Add AWS v11.1.4

### PR DESCRIPTION
**Pull request by marians**
For https://github.com/giantswarm/giantswarm/issues/10143

### CODE REVIEW

File: aws.yaml [hunk 1]
```diff
@@ -1,5 +1,63 @@
...
+    version: 1.2.0
+  components:
+  - name: app-operator
+    version: 1.0.0
+  - name: aws-operator
+    version: 8.2.3
+  - name: cert-operator
+    version: 0.1.0
+  - name: cluster-operator
+    version: 2.1.7
```
Comment (xh3b4sd) [inline aws.yaml:49]:
```suggestion
    version: 2.1.8
```

File: aws.yaml [hunk 2]
```diff
@@ -1,5 +1,63 @@
...
+  - name: net-exporter
+    version: 1.7.0
+  - name: node-exporter
+    componentVersion: 0.18.1
+    version: 1.2.0
+  components:
+  - name: app-operator
+    version: 1.0.0
+  - name: aws-operator
+    version: 8.2.3
```
Comment (xh3b4sd) [inline aws.yaml:45]:
```suggestion
    version: 8.3.0
```

File: release-notes/aws/v11.1.4.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# :zap: Giant Swarm Release v11.1.4 for AWS :zap:
+
+__Note:__ Upgrading to this release from any release prior v11.1.1 will cause a network downtime due to the network-related changes coming with the switch from Calico CNI to AWS CNI.
+
+## aws-operator [v8.2.x](https://github.com/giantswarm/aws-operator/releases/tag/v8.2.x)
```
Comment (xh3b4sd) [inline release-notes/aws/v11.1.4.md:5]:
```suggestion
## aws-operator [v8.3.0](https://github.com/giantswarm/aws-operator/releases/tag/v8.3.0)
```

File: release-notes/aws/v11.1.4.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+- Ensure that Pod subnet IP ranges don't overlap between tenant clusters, to enable VPC peering between tenant clusters in the same installation.
+
+## cert-manager-app [v1.0.7](https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.7)
+
+- Fixed sub-chart resources namespace. ([cert-manager-app#19](https://github.com/giantswarm/cert-manager-app/pull/19), [cert-manager#21](https://github.com/giantswarm/cert-manager-app/pull/21)).
+
+## chart-operator [v0.12.4](https://github.com/giantswarm/chart-operator/releases/tag/v0.12.4)
```
Comment (cokiengchiara) [inline release-notes/aws/v11.1.4.md:13]: @rossf7 I said I'd check w you that latest chart-operator gets into the releases. YAML file version is changed and the release notes are here - that's all I can check. Seems ok for me. Is that good enough for AWS 11.1.4? :)

Comment (cokiengchiara) [reply release-notes/aws/v11.1.4.md:13]: Oh. I saw your conversation with Marian here. https://github.com/giantswarm/giantswarm/issues/10019#issuecomment-615066947 Closing comment.

File: aws.yaml [hunk 5]
```diff
@@ -1,5 +1,63 @@
...
+  name: v11.1.4
+spec:
+  state: wip
+  date: 2020-04-19T11:00:00Z
+  apps:
+  - name: cert-exporter
+    version: 1.2.1
+  - name: cert-manager
+    componentVersion: 0.9.0
+    version: 1.0.7
```
Comment (marians) [inline aws.yaml:15]: @cokiengchiara Is the `componentVersion` correct here?

**Comment by marians** [comment]
@corest Please check the kiam-app details. Thanks!

File: release-notes/aws/v11.1.4.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+# :zap: Giant Swarm Release v11.1.4 for AWS :zap:
+
+__Note:__ Upgrading to this release from any release prior v11.1.1 will cause a network downtime due to the network-related changes coming with the switch from Calico CNI to AWS CNI.
+
+## aws-operator [v8.3.0](https://github.com/giantswarm/aws-operator/releases/tag/v8.3.0)
+
+- Ensure that Pod subnet IP ranges don't overlap between tenant clusters, to enable VPC peering between tenant clusters in the same installation.
```
Comment (xh3b4sd) [inline release-notes/aws/v11.1.4.md:7]: We also added the VPC ID thingy in the CR status. Also not ensuring the CRD anymore happened here like in `cluster-operator`, which has a changelog for that in here.

File: release-notes/aws/v11.1.4.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+## cluster-operator [v2.1.8](https://github.com/giantswarm/cert-manager-app/releases/tag/v2.1.8)
```
Comment (xh3b4sd) [inline release-notes/aws/v11.1.4.md:9]:
```suggestion
## cluster-operator [v2.1.8](https://github.com/giantswarm/cluster-operator/releases/tag/v2.1.8)
```

File: release-notes/aws/v11.1.4.md (hunk 6 above)
Comment (marians) [reply release-notes/aws/v11.1.4.md]: added

**Pull request merged**